internal/pkg/cli/pipeline_deploy.go

```diff
@@ -308,14 +308,16 @@ func (o *updatePipelineOpts) RecommendedActions() []string {
 
 // BuildPipelineUpdateCmd build the command for deploying a new pipeline or updating an existing pipeline.
 func buildPipelineUpdateCmd() *cobra.Command {
-	vars := updatePipelineVars{}
+	vars := deployPipelineVars{}
 	cmd := &cobra.Command{
-		Use:   "update",
-		Short: "Deploys a pipeline for the services in your workspace.",
-		Long:  `Deploys a pipeline for the services in your workspace, using the environments associated with the application.`,
+		Use:     "deploy",
+		Aliases: []string{"update"},
```

Comment:
Do you know if this information gets shown in the help menu or is it hidden?

Reply:
It's hidden!
```
% copilot pipeline deploy --help
Deploys a pipeline for the services in your workspace, using the environments associated with the application.

Usage
  copilot pipeline deploy [flags]

Flags
  -a, --app string   Name of the application. (default "buildspec")
  -h, --help         help for deploy
      --yes          Skips confirmation prompt.

Examples
  Deploys a pipeline for the services and jobs in your workspace.
  `$ copilot pipeline deploy`
```